mixlib-log.gemspec

```diff
@@ -11,7 +11,8 @@ Gem::Specification.new do |gem|
   gem.authors = ["Chef Software, Inc."]
   gem.files = %w{LICENSE} + Dir.glob("lib/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) }
   gem.require_paths = ["lib"]
-  gem.required_ruby_version = ">= 2.7"
+  gem.required_ruby_version = ">= 3.1"
 
-  gem.add_dependency "ffi", "< 1.17.0"
+  # 1.17.1 is broken, see: https://github.com/ffi/ffi/issues/1139
```

Comment:
1.17.1 is not actually broken, but it requires glibc 2.29 for the precompiled gems. See the explanation in https://github.com/ffi/ffi/issues/1139#issuecomment-2632381142.